lib/kubernetes-deploy/render_task.rb

```diff
@@ -69,8 +69,12 @@ def render_filename(filename, stream)
       file_content = File.read(File.join(@template_dir, filename))
       rendered_content = @renderer.render_template(filename, file_content)
       YAML.load_stream(rendered_content, "<rendered> #{filename}") do |doc|
-        stream.puts YAML.dump(doc)
+        YAML.dump(doc)
```

Comment:
Why do we still need to do this? I think it is the loading that will catch syntax errors, and would expect `parse_stream` below to do so as effectively.